lib/spanner/bundle/config_validator.ex

```diff
@@ -51,6 +45,32 @@ defmodule Spanner.Bundle.ConfigValidator do
     validate_commands!(btype, commands)
   end
 
+  # Validate bundle Type
+  defp validate_bundle_type!(json, field) do
```

Comment:
While this works, the raise / rescue / raise pattern surrounding a `!` function call (in addition to being very non-idiomatic Elixir) suggests that a change to `JsonNavigator` would be more appropriate. 

At first glance, we seem to want a version of `get!` that takes a list of acceptable values, as well as a default if none is found. Or perhaps a new function altogether that has just this functionality; that may be the better approach in terms of readability, etc.